scripts/install/MapPasswordControlFeatureFlag.php

```diff
@@ -37,6 +40,16 @@ public function __invoke($params)
             SectionVisibilityFilter::OPTION_FEATURE_FLAG_SECTIONS,
             $featureFlagSections
         );
+        /** @var core_kernel_classes_Resource $ffResource */
+        $ffResource = $this->getServiceManager()
+            ->get(Ontology::SERVICE_ID)
+            ->getResource('http://www.tao.lu/Ontologies/TAO.rdf#featureFlags');
+
+        $ffProperty = new core_kernel_classes_Property(
+            'http://www.tao.lu/Ontologies/TAO.rdf#featureFlags_FEATURE_FLAG_PASSWORD_CHANGE_AVAILABLE'
```

Comment:
After using that https://github.com/oat-sa/extension-tao-backoffice/pull/167/files#r1252856843
you just need to use `FEATURE_FLAG_PASSWORD_CHANGE_AVAILABLE` here instead.